commands/backstop-remote.js

```diff
@@ -3,17 +3,19 @@ const backstopjs = require('backstopjs');
 
 module.exports = {
   name: 'backstop:remote',
+
   aliases: ['backstop-remote'],
+
   availableOptions: [
-    { name: 'filter', type: String, aliases: ['f'], default: ''},
-    { name: 'config', type: String, aliases: ['c'], default: './backstop.js' }
+    { name: 'filter', type: String, aliases: ['f'], default: '' },
+    { name: 'config', type: String, aliases: ['c'], default: './backstop.js' },
+    { name: 'i', type: Boolean },
```

Comment:
So, `i` here is not really a parameter to remote.  Neither is `filter` for that matter (this line probably got copy-pasted from another command that takes `filter`)

Config is a parameter tho.  we should leave that.